plugins/woocommerce-admin/client/customize-store/assembler-hub/sidebar/sidebar-navigation-screen-homepage.tsx

```diff
@@ -31,10 +31,7 @@ export const SidebarNavigationScreenHomepage = () => {
 	const [ blocks, , onChange ] = useEditorBlocks();
 	const onClickPattern = useCallback(
 		( _pattern, selectedBlocks ) => {
-			onChange(
-				[ blocks[ 0 ], ...selectedBlocks, blocks[ blocks.length - 1 ] ],
-				{ selection: {} }
-			);
+			onChange( selectedBlocks, { selection: {} } );
```

Comment:
Hey @ilyasfoo My understanding is on this screen, we show the homepage templates without header/footer.

@verofasulo Could you please confirm that when users select a homepage on this screen will replace the header/footer together? Thank you! 🙏 

<img width="1280" alt="Page list (3)" src="https://github.com/woocommerce/woocommerce/assets/4344253/b0b39062-0bec-4a3a-878e-7ad6fe413373">

Reply:
Attach the current changes's recording.

https://github.com/woocommerce/woocommerce/assets/4344253/8ca2fa5b-581a-4325-85d0-a4df26c22219

Reply:
Ah, I can't believe I missed this fact 🤦 

I'll fix it next week

Reply:
@chihsuan I've fixed it in https://github.com/woocommerce/woocommerce/pull/40363/commits/b1ce8b108f41f7335c6dbc4c0987b5be276378f3d added tests in https://github.com/woocommerce/woocommerce/pull/40363/commits/68ce0211141121819882cbfb53613047fd7efd49. Please take another look! 🙏